components/ide/gha-update-image/lib/code-pin-version.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+import { $ } from "bun";
+import {
+    getIDEVersionOfImage,
+    getLatestInstallerVersions,
+    pathToConfigmap,
+    readIDEConfigmapJson,
+    readWorkspaceYaml,
+} from "./common";
+
+$.nothrow();
+
+const ideConfigmapInfo = await readIDEConfigmapJson();
+const ideConfigmapJson = ideConfigmapInfo.parsedObj;
+const ideConfigmapJsonObj = ideConfigmapInfo.rawObj;
+const workspaceYaml = await readWorkspaceYaml().then((d) => d.parsedObj);
+
+export async function updateCodeIDEConfigJson() {
```

Comment:
nit: should this be `updateCodeIDEConfigMapJson` instead to keep consistent with `readIDEConfigmapJson`?